cli/src/deno.rs

```diff
@@ -230,13 +225,9 @@ fn transpile(
 
 /// Load the internal Phylum API module
 fn phylum_module() -> Result<ModuleSource> {
-    let module_url = "deno:phylum";
-    let code = transpile(module_url, EXTENSION_API, MediaType::TypeScript)?;
-
-    Ok(ModuleSource {
-        module_url_specified: module_url.into(),
-        module_url_found: module_url.into(),
-        module_type: ModuleType::JavaScript,
-        code: code.into(),
-    })
+    let module_url = ModuleSpecifier::parse("deno:phylum")
+        .expect("`deno:phylum` is not a valid module specifier");
```

Comment:
Can this actually fail? I might prefer unwrap here because I wouldn't expect a literal to ever fail parsing?